### ISSUE TITLE
Mu v1.1.2 macOS Mojave Crash Report

### ISSUE THREAD

**Issue by carlosperate** [mu-editor/crash-reports#350]
**REMEMBER TO ATTACH YOUR LOG FILE**

Version: 1.1.2
Locale: en_GB
Platform: Darwin 18.5.0 Darwin Kernel Version 18.5.0: Mon Mar 11 22:26:31 PDT 2019; root:xnu-4903.251.3~5/RELEASE_X86_64 x86_64

```
/Downloads/Mu Editor.app/Contents/Resources/Python/lib/python3.8/site-packages/mu/virtual_environment.py", line 947, in install_baseline_packages
    self.install_from_zipped_wheels(zipped_wheels_filepath)
  File "/Users/carlos/Downloads/Mu Editor.app/Contents/Resources/Python/lib/python3.8/site-packages/mu/virtual_environment.py", line 921, in install_from_zipped_wheels
    self.pip.install(wheel, deps=False, index=False)
  File "/Users/carlos/Downloads/Mu Editor.app/Contents/Resources/Python/lib/python3.8/site-packages/mu/virtual_environment.py", line 295, in install
    return self.run(
  File "/Users/carlos/Downloads/Mu Editor.app/Contents/Resources/Python/lib/python3.8/site-packages/mu/virtual_environment.py", line 271, in run
    result = self.process.run_blocking(
  File "/Users/carlos/Downloads/Mu Editor.app/Contents/Resources/Python/lib/python3.8/site-packages/mu/virtual_environment.py", line 147, in run_blocking
    return self.wait(wait_for_s=wait_for_s)
  File "/Users/carlos/Downloads/Mu Editor.app/Contents/Resources/Python/lib/python3.8/site-packages/mu/virtual_environment.py", line 211, in wait
    raise VirtualEnvironmentError(
mu.virtual_environment.VirtualEnvironmentError: Process finished but with error code 1:
ERROR: pyzmq-24.0.1-cp38-cp38-macosx_10_15_universal2.whl is not a supported wheel on this platform.

```

**Comment by carlosperate** [comment]
The trick to remove the wheels zip file and let Mu redownload them works, so I've uploaded the new zip from Mojave:
https://we.tl/t-rphC380Opc

We can use this to check if, after a fix, the same wheels are downloaded in newer macOS versions.

**Comment by carlosperate** [comment]
Looks like the only difference between the packaged zip file and the one generated in Mojave is the `pyzmq` wheel:
<img width="1542" alt="image" src="https://user-images.githubusercontent.com/4189262/200806421-32e023bd-84d6-4a61-9e68-8d5cd856d486.png">

Looking at the other wheels I don't expet any other issues wither with older macOS versions either as all are compatible with 10.10+ (Yosemite) and PyQt5 is 10.12+ (Sierra).

**Comment by carlosperate** [comment]
This has been resolved in v1.2.0 🎉
